azure-mgmt-network/src/main/java/com/microsoft/azure/management/network/TrafficAnalyticsConfigurationProperties.java

```diff
@@ -23,19 +23,19 @@
     /**
      * The resource guid of the attached workspace.
      */
-    @JsonProperty(value = "workspaceId", required = true)
+    @JsonProperty(value = "workspaceId")
```

Comment:
Please document this changes.

Reply:
Swagger network 2019-07 would have this fix.